### ISSUE TITLE
Tests in Microsoft.R.Components.Test get stuck when running all of them

### ISSUE THREAD

**Issue by huguesv** [Microsoft/RTVS#2464]
Looks like they run 4 at a time, and right now it's stuck with those 4:
```
Test Name:	PackageManagerIntegrationTest.GetPackageLockStateLockByRSession
Test Name:	RHistoryIntegrationTest.InteractiveWindowIntegration01
Test Name:	RPlotIntegrationTest.SomeCommandsEnabledForMiddlePlot
Test Name:	RPlotIntegrationUITest.ResizePlotError
```

**Comment by MikhailArkhipov** [comment]
This is b/c of the object disposed exception (which I fixed) as well as #2481

**Comment by MikhailArkhipov** [comment]
Closing in favor of #2481
